mesh/mesh-io-tcpserver.c

```diff
@@ -269,6 +315,112 @@ static void tcpserver_io_init_done(void *user_data)
 	mesh_io->pvt->ready_callback(mesh_io->pvt->user_data, true);
 }
 
+static unsigned int tls_psk_server_cb(SSL * ssl, const char *identity,
+                                      unsigned char *psk,
+                                      unsigned int max_psk_len)
+{
+	const char * k1_info = "idpsk";
```

Comment:
I'd write this as `static const uint8_t k1_info[] = {'i', 'd', 'p', 's', 'k'};` and then `sizeof(k1_info)`